qube/qube_cli.py

```diff
@@ -142,45 +161,42 @@ def sync(project_dir, set_token, pat, username, check_update) -> None:
     if set_token:
         try:
             project_data = load_yaml_file(f'{project_dir}/.qube.yml')
-            repo_owner = project_data['github_username']
-            # if project is an orga repo, pass orga name as username
-            if project_data['is_github_repo'] and project_data['is_github_orga']:
```

Comment:
@sven1103 Why did we even pass the new repo_owner thingy here?
It was not used at all.

Reply:
huh, yes indeed 🤦